packages/aws-cdk-lib/custom-resources/README.md

```diff
@@ -607,9 +607,16 @@ new cr.AwsCustomResource(this, 'ListObjects', {
 Note that even if you restrict the output of your custom resource you can still use any
 path in `PhysicalResourceId.fromResponse()`.
 
-### Custom Resource Examples
+### Custom Resource Logging
```

Comment:
Lets add an example Response Object in these docs

Reply:
Good call out. This has been updated.